alacritty_terminal/src/tty/windows/mod.rs

```diff
@@ -124,12 +123,12 @@ impl OnResize for Pty {
 }
 
 fn cmdline(config: &PtyConfig) -> String {
-    let default_shell = Program::Just("powershell".to_owned());
+    let default_shell = Shell { program: "powershell".to_owned(), args: Vec::new() };
     let shell = config.shell.as_ref().unwrap_or(&default_shell);
 
-    once(shell.program())
-        .chain(shell.args().iter().map(|a| a.as_ref()))
-        .collect::<Vec<_>>()
+    once(shell.program.as_str())
```

Comment:
```suggestion
    once(&shell.program)
```

I'd assume this doesn't work, but if it did it would be better.

Reply:
It doesn't.